pkg/api/resource_test.go

```diff
@@ -0,0 +1,178 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package api
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/davecgh/go-spew/spew"
+	"k8s.io/apimachinery/pkg/api/resource"
+)
+
+func TestResourceNameString(t *testing.T) {
+	testCases := []struct {
+		resourceName       ResourceName
+		resourceNameString string
+	}{
+		{
+			resourceName:       ResourceName("secret"),
+			resourceNameString: "secret",
+		},
+	}
+
+	for _, tc := range testCases {
+		if tc.resourceName.String() != tc.resourceNameString {
+			t.Errorf("expected ResourceName string to be '%v', but got '%v' instead", tc.resourceNameString, tc.resourceName.String())
+		}
+	}
+}
+
+func TestResourceListCpu(t *testing.T) {
+	testCases := []struct {
+		description      string
+		resourceList     ResourceList
+		resourceQuantity resource.Quantity
+	}{
+		{
+			description:      "ResourceCPU is present in ResourceList",
+			resourceList:     ResourceList{ResourceCPU: resource.MustParse("100m")},
+			resourceQuantity: resource.MustParse("100m"),
```

Comment:
please call the expected outcome `expected`. It would make it much easier to understand what the test is actually doing.

Reply:
done, changed in all the test structs in the file